docs/source/development/pack_testing.rst

```diff
@@ -0,0 +1,98 @@
+:orphan:
+
+Pack Testing
+============
+
+This section includes information on pack testing - where to put the tests,
+how to write the tests, mock classes which can be used to make testing
+easier, etc.
+
+Test File Locations and Names
+-----------------------------
+
+All the test files should go into ``<pack name>/tests/`` directory. If tests
+include any fixtures, they should be put in the ``<pack name>/tests/fixtures/``
+directory.
+
+Test files should follow the following naming conventions:
+
+* ``test_action_<action name>.py`` for action tests. For example, if the action
```

Comment:
`test_*.py` I am on board with.

Since the other naming convention is not a requirement this works for me. I guess we could tweak the language in the docs from "should follow the following ..." to "are recommended to follow ...". Since that will make the `test_*.py` part awkward to explain I do not insist.